Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,23 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+
+// part 1 mapping to hours
+
+const hours = tasks.map(tasks => tasks.duration / 60);
```

Comment:
I would like to ask you to remove all the `console.log` statements as we don't really need them once we have got working code ready.

Reply:
it is done my friend, removed the console.log... and I left the last one commented because is part of the homework, because it resume the whole amount to two decimals.